fairlearn/__init__.py

```diff
@@ -1,4 +1,17 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
 __name__ = "fairlearn"
 __version__ = "0.3.0-alpha"
 
 _KW_SENSITIVE_FEATURES = "sensitive_features"
+
+
+# Setup logging infrustructure
+import logging  # noqa: E402
+import os  # noqa: E402
```

Comment:
Why the flake8 suppression?

Reply:
E402 module level import not at top of file

I don't want to put it there because this file is mainly for the version